app/src/main/java/io/pumpkinz/pumpkinreader/NewsDetailFragment.java

```diff
@@ -77,6 +77,10 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
         newsDetail = (RecyclerView) view.findViewById(R.id.news_detail);
+        newsDetail.getItemAnimator().setRemoveDuration(100);
+        newsDetail.getItemAnimator().setChangeDuration(0);
```

Comment:
Why is change duration the only one with 0 seconds duration, not 100 like the others?